### ISSUE TITLE
Why classifier outputs 1001 logits instead of 1000?

### ISSUE THREAD

**Issue by sayakpaul** [tensorflow/hub#574]
Hi,

I am referring to this tutorial: https://www.tensorflow.org/tutorials/images/transfer_learning_with_hub. As you can see below the classifier outputs a 1001-d vector of logits. Isn't it supposed to return a 1000-d vector?

![image](https://user-images.githubusercontent.com/22957388/80863466-30373c80-8c9a-11ea-895c-5ba59a7be6a3.png)

**Comment by gowthamkpr** [comment]
As you can see from the [source code](https://github.com/tensorflow/models/blob/394baa9f21424d3522ccfbdcee8acd3840891ff6/research/slim/nets/mobilenet/mobilenet_v2.py#L107) of Mobilenet_v2 itself the num_classes is 1001.

**Comment by sayakpaul** [comment]
That's okay but I wanted to understand why that might be the case.

**Comment by arnoegw** [comment]
If you check the list of labels linked from https://tfhub.dev/google/tf2-preview/mobilenet_v2/classification/2, you'll see that class 0 is "background", and real object classes start with index 1. This is a common technique for image classifiers that can be applied to box predictions i object detection.

**Comment by sayakpaul** [comment]
Got it. Some clarification regarding this in the documentation itself would have been helpful.

**Comment by arnoegw** [comment]
The documentation for our current suite of ImageNet classification models has been updated. See, e.g., https://tfhub.dev/google/imagenet/mobilenet_v2_100_224/classification/4
